pkg/kubectl/rolebinding.go

```diff
@@ -147,7 +147,7 @@ func (s RoleBindingGeneratorV1) StructuredGenerate() (runtime.Object, error) {
 	}
 	for _, sa := range sets.NewString(s.ServiceAccounts...).List() {
 		tokens := strings.Split(sa, ":")
-		if len(tokens) != 2 {
+		if len(tokens) != 2 || tokens[0] == "" || tokens[1] == "" {
```

Comment:
namespace is allowed to be empty here since that indicates "use the local namespace"